src/components/Calculator.js

```diff
@@ -0,0 +1,28 @@
+function Calculator() {
+  return (
+    <div className="calc">
+      <span className="keys4">0</span>
+      <span className="key1">AC</span>
+      <span className="key1">+/-</span>
+      <span className="key1">%</span>
+      <span className="key1b">÷</span>
+      <span className="key1">7</span>
+      <span className="key1">8</span>
+      <span className="key1">9</span>
+      <span className="key1b">*</span>
+      <span className="key1">4</span>
+      <span className="key1">5</span>
+      <span className="key1">6</span>
+      <span className="key1b">-</span>
+      <span className="key1">1</span>
+      <span className="key1">2</span>
+      <span className="key1">3</span>
+      <span className="key1b">+</span>
```

Comment:
- [ ] According to the project's requirements, the `<Calculator />` component should have at least one child component. You may create a reusable `<Button />` component for the buttons.
- [ ] Please let's use semantic HTML. Calculators have `buttons`, so kindly replace the `<span>` with `<button>` as it improves accessibility, readability, and maintainability,